src/components/pages/strategies/Pending/PendingTable.tsx

```diff
@@ -87,7 +87,7 @@ export const PendingTable = memo(function PendingTable({
                 <TableCell>{strategy.created.toLocaleString()}</TableCell>
                 <TableCell>
                   {strategy.quoteToken && strategy.baseToken
-                    ? `${strategy.quoteToken?.symbol} - ${strategy.baseToken?.symbol}`
+                    ? `${strategy.baseToken?.symbol} - ${strategy.quoteToken?.symbol}`
```

Comment:
Nice! This always confuses me that 600 WETH-OWL (or WETH/OWL) actually means 600 OWL per 1 WETH :exploding_head: